### ISSUE TITLE
Flan-T5

### ISSUE THREAD

**Issue by Bachstelze** [deep-floyd/IF#75]
Can we use [FLAN-T5](https://huggingface.co/google/flan-t5-base) as a language model?
Those FLAN models can represent English and other languages significantly better in our tests.
"If you already know T5, FLAN-T5 is just better at everything. For the same number of parameters, these models have been fine-tuned on more than 1000 additional tasks covering also more languages."

**Comment by phalexo** [comment]
Wouldn't you have to retrain text to image models because the representations are different?

**Comment by darkman111a** [comment]
Yes, could expertiment with finetuning once you swap it out. I'm 1000% sure FLAN-T5 would result in higher fidelity output, better composition, way better spatial awareness. I think "tango model" kind of validates this.
